.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```

Comment:
Not that it is an issue, but where are these files coming from?  It looks like you're using intellij as a code editor?  Have you learned about gitignore yet?